### DISCUSSION TITLE
Django not deleting session id in mongoDB database when logging out

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#3863] [Q&A]
I have integrated the MongoDB database with CVAT.
I get "Could not login" Error when I try to login **second time** into CVAT portal.
I get below error in browser console 
"FAILED SQL: INSERT INTO "auth_user_groups" ("user_id", "group_id") VALUES (%(0)s, %(1)s)" 
I know that this is because django has not deleted the session id in database in auth_user_groups collection for first time login-logout. 
How can I fix this issue and make sure I will be able login-logout multiple times without having to manually delete session id in mongoDB ,auth_user_groups collection.

**Comment by azhavoro** [comment]
Hi, we have never tried to use a database backend that Django does not natively support, and we have no plans to support MongoDB in the future. CVAT doesn't have the `logout` logic implementation and use  [`Dgango rest auth`](https://django-rest-auth.readthedocs.io/en/latest/#) package for this  https://github.com/openvinotoolkit/cvat/blob/6e49f1b36b78fddbfd88646132ddde0b3e407bf1/cvat/apps/authentication/urls.py#L17. So I would suggest asking the authors of this package about your issue.
P.S. Why postgres doesnt meet your requirements and you want to use MongoDB?
